common-algo.c

```diff
@@ -82,11 +82,9 @@ static const struct dropbear_cipher dropbear_3des =
 const struct dropbear_cipher dropbear_nocipher =
 	{NULL, 16, 8}; 
 
-/* A few void* s are required to silence warnings
- * about the symmetric_CBC vs symmetric_CTR cipher_state pointer */
 #if DROPBEAR_ENABLE_CBC_MODE
 const struct dropbear_cipher_mode dropbear_mode_cbc =
-	{(void*)cbc_start, (void*)cbc_encrypt, (void*)cbc_decrypt};
+	{cbc_start, cbc_encrypt, cbc_decrypt};
```

Comment:
This change isn't valid, cbc_decrypt and ctr_decrypt actually have different signatures. https://travis-ci.org/mkj/dropbear/jobs/238374212